android/assets/jsons/translations/French.properties

```diff
@@ -2703,51 +2703,51 @@ Military Tradition = Tradition militaire
 Military units gain 50% more Experience from combat = Expérience gagnée en combat +50% pour les unités militaires
 Military Caste = Caste militaire
  # Requires translation!
-[stats] in all cities with a garrison = 
+[stats] in all cities with a garrison = [stats] dans chaques villes dotées d'une garnison 
 Professional Army = Armée de métier
 Honor Complete = Honneur complété
 Gain gold for each unit killed = Gain d'or à chaque unité tuée
 Honor = Honneur
  # Requires translation!
-+25% bonus vs Barbarians = 
++25% bonus vs Barbarians = +25% bonus contres les barbares 
  # Requires translation!
-Gain Culture when you kill a barbarian unit = 
+Gain Culture when you kill a barbarian unit = Gain de culture lors que vous tuez un barbares 
  # Requires translation!
-Notified of new Barbarian encampments = 
+Notified of new Barbarian encampments = Notification d'un nouveau camp de barbares
 
 Organized Religion = Religion organisée
 Mandate Of Heaven = Mandat du Ciel
 50% of excess happiness added to culture towards policies = 50% du bonheur excédentaire est ajouté en Culture
 Theocracy = Théocratie
  # Requires translation!
-+[amount]% [stat] from every [building] = 
++[amount]% [stat] from every [building] = +[amount]% [stat] de chaque [building] 
 Reformation = Réforme
  # Requires translation!
-+33% culture in all cities with a world wonder = 
++33% culture in all cities with a world wonder = +33% de culture dans toute les villes dotées d'une Merveille 
 Free Religion = Tolérance religieuse
 Piety Complete = Piété complète
 Classical era = Âge classique
 Piety = Piété
  # Requires translation!
-+[amount]% Production when constructing [stat] buildings = 
++[amount]% Production when constructing [stat] buildings = +[amount]% de production lors de la construction [stats] de bâtiments 
 
 Trade Unions = Unions commerciales
  # Requires translation!
-Maintenance on roads & railroads reduced by 33% = 
+Maintenance on roads & railroads reduced by 33% = Maintenance des routes et des chemins de fer réduits de 33%
 Mercantilism = Mercantilisme
 Entrepreneurship = Entreprenariat
 Patronage = Patronat
  # Requires translation!
-Cost of purchasing [stat] buildings reduced by [amount]% = 
+Cost of purchasing [stat] buildings reduced by [amount]% = Coût d'achat [stats] des bâtiments réduits de [amount]%
```

Comment:
`Cost of purchasing [stat] buildings reduced by [amount]% = Coût d'achat [stat] des bâtiments réduits de [amount]%
`